pkg/dmaasbackup/schedule.go

```diff
@@ -72,12 +77,84 @@ func (d *dmaasBackup) generateBackupName(dbkp v1alpha1.DMaaSBackup) string {
 // updateScheduleInfo checks for relevant velero schedule/backup and
 // update the dbkp with schedule/backup details
 func (d *dmaasBackup) updateScheduleInfo(dbkp *v1alpha1.DMaaSBackup) error {
-	// TODO
+	var (
+		schedule *velerov1api.Schedule
+		bkp      *velerov1api.Backup
+		err      error
+	)
+
+	// check if dbkp is having config for non-scheduled backup
+	if yes := isSchedule(*dbkp); !yes {
```

Comment:
this code is also not required.. its handled in processNonperiodicConfigSchedule

Reply:
moreover there can be cases of periodic full backup of just backups (not schedules)